### PR TITLE
EntityList row selection goes to details view

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

**Comment by pmbohm** [comment]
https://github.com/aodn/backlog/issues/2635

File: ui/src/components/data-entities/EntityList.js [hunk 1]
```diff
@@ -108,6 +108,19 @@ const nonGenericEntities = {
...
+const gotoDetailsView = (event) => {
+  if (event.node.isSelected() &&
+    event.colDef.field !== 'Links' &&
+    event.node.data._links )
+  {
+    const hrefSplit = event.node.data._links.self.href.split('/');
+    const id = hrefSplit.pop();
+    const ent = hrefSplit.pop();
+    window.location = '/view/' + ent + '/' + id;
+  }
```
Comment (utas-tajona) [inline ui/src/components/data-entities/EntityList.js:121]: window.location need to be avoid. use the hook location instead

Comment (utas-tajona) [reply ui/src/components/data-entities/EntityList.js:121]:
```
const history = useHistory();
.....

history.push('/view/' +ent + '/' + id);
```

**Review by utas-tajona — CHANGES_REQUESTED** [review]

**Comment by ccmoloney** [comment]
@utas-tajona Can you please merge this if you are happy with the updates?

**Pull request merged**